tlscanary/template/js/log_handler.js

```diff
@@ -0,0 +1,75 @@
+
+var type_map = {};
+var visible_columns = {};
+var final_data = {};
+var transform_rules;
+
+function test(arg)
```

Comment:
I'll mark this once, but it's all over the code: The formatting convention in the later JS code is

```javascript
function foo() {
    return;
}

function baz() {
    return;
}
```

with the opening curly trailing on the definition line and one empty line between function definitions.

I'd suggest to globally adapt https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style with long lines for any new code.